tests/config_input/sn.yaml

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2012-2022 by the GalSim developers team on GitHub
+# https://github.com/GalSim-developers
+#
+# This file is part of GalSim: The modular galaxy image simulation toolkit.
+# https://github.com/GalSim-developers/GalSim
+#
+# GalSim is free software: redistribution and use in source and binary forms,
+# with or without modification, are permitted provided that the following
+# conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice, this
+#    list of conditions, and the disclaimer given in the accompanying LICENSE
+#    file.
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions, and the disclaimer given in the documentation
+#    and/or other materials provided with the distribution.
+#
+
+
+modules:
+    - numpy
```

Comment:
Is this needed for the `np.exp(-time/50)` bit down below?  I know `import numpy as np` is canonical, but I'm a little surprised `np.exp` works instead of `numpy.exp` given this line.  Just curious how this works.

Reply:
No.  I'm not sure why this is here.  I think it must be a relic from something that since changed.  I removed it.

But to answer your question about np.exp, we automatically import numpy, both as numpy and as np, so those are always available without anything special on the part of the user.